portlets/src/main/webapp/vue-app/realizations/components/Realizations.vue

```diff
@@ -202,13 +209,15 @@ export default {
         });
     },
     getRealizations() {
-      return this.$realizationsServices.getAllRealizations(this.fromDate, this.toDate, this.sortBy, this.sortDescending, this.offset, this.limit + 1)
+      this.returnType = 'json';
+      return this.$realizationsServices.getAllRealizations(this.fromDate, this.toDate,this.earnerId, this.sortBy, this.sortDescending, this.offset, this.limit + 1, this.returnType)
         .then(realizations => {
           this.realizations = realizations || [];
         });
     },
     exportFile() {
-      return this.$realizationsServices.exportFile(this.fromDate, this.toDate);
+      this.returnType = 'xls';
+      return this.$realizationsServices.exportFile(this.fromDate, this.toDate, this.earnerId, this.returnType);
```

Comment:
no need to use a data for `this.returnType`
```suggestion
      return this.$realizationsServices.exportFile(this.fromDate, this.toDate, this.earnerId, 'xls');
```